src/UnityMvvmToolkit.Core/Internal/ObjectHandlers/ObjectWrapperHandler.cs

```diff
@@ -110,7 +110,7 @@ public ObjectWrapperHandler(ValueConverterHandler valueConverterHandler)
             var commandId =
                 HashCodeHelper.GetCommandWrapperId(context.GetType(), commandValueType, bindingData.PropertyName);
 
-            if (_commandWrappers.TryGetValue(commandId, out var commandWrapper))
+            if (_commandWrappers.TryGetValue(commandId, out var commandWrapper) && String.IsNullOrEmpty(bindingData.ConverterName))
```

Comment:
If you change the `if` condition like this. The objects from the pool will never be used. For commands, this check is useless because the converter will be used anyway.